waku/waku.md

```diff
@@ -270,6 +272,48 @@ This feature will likely stop being experimental in v1.
 
 It is currently bounded to a maximum of 1000 topics. If you are interested in more topics than that, this is currently underspecified and likely requires updating it. The constant is subject to change.
 
+**Message Confirmations**
+
+Message confirmations tell a node that a message originating from it has been received by its peers, allowing a node to know whether a message has or has not been received.
+
+A node MAY send a message confirmation for any batch of messages received with a packet Messages Code (`0x01`).
+
+A message confirmation is sent using Batch Acknowledge packet (`0x0b`) or Message Response packet (`0x0c`).
+
+The Batch Acknowledge packet is followed by a keccak256 hash of the envelopes batch data (raw bytes).
```

Comment:
Probably don't need to be too anal about describing data format here, that's what the ABNF is for